<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddOrEditPracWork</class>
 <widget class="QDialog" name="AddOrEditPracWork">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_error">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:red;</string>
     </property>
     <property name="text">
      <string>Заполните обязательные поля</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_group">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Выберите группу*</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QComboBox" name="comboBox_grupa">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_student">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Выберите студента*</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QComboBox" name="comboBox_student">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_tip_prac">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Выберите тип практики*</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QComboBox" name="comboBox_typePrac">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_semestr">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Выберите семместр*</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QComboBox" name="comboBox_semestr">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_rukKaf">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Выберите руководителя от кафедры</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QComboBox" name="comboBox_rucKaf">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_rucPred">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Введите руководителя от предприятия</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLineEdit" name="lineEdit_rukPredpr">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_MestoProhoj">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Место прохождения*</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLineEdit" name="lineEdit_MestoProhoj">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="label_Ocenka">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Оценка</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QLineEdit" name="lineEdit_ocenka">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="label_datas">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Выберите дату</string>
     </property>
    </widget>
   </item>
   <item row="18" column="0">
    <widget class="QDateEdit" name="dateEdit">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="displayFormat">
      <string>dd MMMM yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="19" column="0">
    <widget class="QLabel" name="label_otziv">
     <property name="text">
      <string>Отзыв</string>
     </property>
    </widget>
   </item>
   <item row="20" column="0">
    <widget class="QPlainTextEdit" name="plainTextEdit_otziv">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="21" column="0">
    <widget class="QLabel" name="label_nomer_prikaza">
     <property name="text">
      <string>Номер приказа</string>
     </property>
    </widget>
   </item>
   <item row="22" column="0">
    <widget class="QLineEdit" name="lineEdit_nomPrikaza"/>
   </item>
   <item row="23" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="pushButton_ok">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
      <property name="icon">
       <iconset resource="../resourses.qrc">
        <normaloff>:/img/img/ok.png</normaloff>:/img/img/ok.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Отмена</string>
      </property>
      <property name="icon">
       <iconset resource="../resourses.qrc">
        <normaloff>:/img/img/cancel.png</normaloff>:/img/img/cancel.png</iconset>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resourses.qrc"/>
 </resources>
 <connections/>
</ui>
